dockers/docker-database/database_config.json.j2

```diff
@@ -53,10 +53,25 @@
             "separator": "|",
             "instance" : "redis"
         },
-        "RESTAPI_DB" : {
+        "GB_ASIC_DB" : {
             "id" : 8,
             "separator": "|",
             "instance" : "redis"
+        },
+        "GB_COUNTERS_DB" : {
+            "id" : 9,
+            "separator": "|",
+            "instance" : "redis"
+        },
+        "GB_FLEX_COUNTER_DB" : {
+            "id" : 10,
+            "separator": "|",
+            "instance" : "redis"
+        },
```

Comment:
let's add this after RESTAPI_DB, do not change the RESTAPI_DB

Reply:
@lguohan - this happened also about a month ago. Another project came in and claimed "8". Problem was, we have other commits (in sonic-swss-common or sonic-swss, I'd need to look) that refer to these values as sonstants.  In the prior event, we did that same thing, renumbered the id based on fact GB already had these alocated (even though hard to find, being in a pull request at the time).

Reply:
Addressed here and in PR https://github.com/Azure/sonic-swss-common/pull/385